docs/cloud-integration.md

```diff
@@ -257,4 +257,5 @@ Here is the documentation on the standard connectors both from Apache and the cl
 * [Amazon EMR File System (EMRFS)](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-fs.html). From Amazon
 * [Google Cloud Storage Connector for Spark and Hadoop](https://cloud.google.com/hadoop/google-cloud-storage-connector). From Google
 * [The Azure Blob Filesystem driver (ABFS)](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-abfs-driver)
+* IBM Cloud Object Storage connector for Apache Spark : [Stocator,](https://github.com/CODAIT/stocator) [IBM Object Storage,](https://www.ibm.com/cloud/object-storage) [how-to-use-connector](https://developer.ibm.com/code/2018/08/16/installing-running-stocator-apache-spark-ibm-cloud-object-storage)
```

Comment:
If this is more or less the equivalent of "S3 connector docs" for AWS, but for the IBM cloud, it could be OK. However I think this doc is a little more about what Spark directly supports, particularly through `hadoop-cloud`. (In any event I think you need to fix the anchors? they have commas in them.) Would this be more appropriate at https://github.com/apache/spark-website/blob/asf-site/third-party-projects.md ? it seems to refer to a third-party integration, not first-party cloud docs.

Reply:
@srowen Thank you very much for your quick response. So here, i was trying to model after how Google Cloud Storage Connector is specified in this section. 
> In any event I think you need to fix the anchors? they have commas in them.

So Sean, here i had three links 1) To the connector 2) IBM cloud storage 3) A devworks ariticle that ties them together. So i had them separated by comma. Should i just remove the commas and have just a space as a separator ?

Reply:
Yes, but I am wondering first whether this is the right place. `hadoop-cloud` and thus Spark doesn't have special support for this connector, and that's what this doc is about.

I'm also just noting that it seemed odd to put the comma within the hyperlinked text.

Reply:
@srowen Thanks.. I am not a 100% sure about whether this is the right place :-). Could you please help me understand how `Google Cloud Storage Connector for Spark and Hadoop` is placed here ? When i click here and navigate to the connector link .. i end up here `https://github.com/GoogleCloudPlatform/bigdata-interop/tree/master/gcs which is the connector for Google cloud storage which i thought is similar to the stocator link ?

Reply:
Actually I think this place is fine, after re-reading the doc. It is a more general reference. I would just fix the links a bit. `[...](...),` not `[...,](...)`

Reply:
@srowen Thank you. I have updated per your advice.